docs/release-notes.md

```diff
@@ -10,6 +10,42 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and fixes made in different versions.
 
+## March 2022
+
+### Version 0.84.0
+
+#### Additions
+
+##### Semgrep CLI lists supported languages
+
+Semgrep CLI now includes `--show-supported-languages` flag to display the list of languages supported by semgrep. Thanks to John Wu for this contribution! ([#4754](https://github.com/returntocorp/semgrep/pull/4754))
```

Comment:
In "... display the list of languages supported by semgrep"

We should capitalize Semgrep unless we're referring to the command line  `semgrep` in which case use backticks.

Reply:
Oh yes, of course... how could it slip through... thank you. :D